### DISCUSSION TITLE
Schedule meeting to a specific bbb  endpoint.

### DISCUSSION THREAD

**Discussion by naimarshad** [blindsidenetworks/scalelite#1023] [Q&A]
Is it possible to schedule/start meetings from a specific user/domain to always go to specific bbb server. So if I can explain with an example it would be that user1@org.com will be sent to server1 and user1@org.net will be sent to server2.

Regards,
Naeem

**Comment by JeanPluzo** [comment]
Hi,

currently it's not possible to target specific servers.
However, since v. 1.5, SL has multitenancy. With this you could make 1, 2 or more tenants and enable specific users to a tenant through your preferred frontend (Greenlight, PILOS, etc).
This is just in theory. I haven't tried it myself.

Regards,
J.

**Reply by naimarshad** [reply]
Hi,
Thanks for your prompt reply, so according to my situation and considering the theory let's say I will create my setup like this

1. Tenants in SL

- tenant1
- tenant2

GL pointing to SL

- bbb-instance1
- bbb-instance2

Then based on my users domain let's say user from org.com can be schedule using tenant1 on bbb-instance1 and users from org.net using tenant2 can be schedule on bbb-instance2?

I hope I understood it right and explained well enough.

Regards,

**Reply by JeanPluzo** [reply]
Hi again,
yes something like this should work. Again, I haven't tried it myself.
Quick note: GL can't point to different domains/bbb servers at the same time. It might be better to have a GL instance for each BBB instance (in your example gl1->bbb-instance1, gl2->bbb-instance2). This way you won't get anything mixed up. Downside is that the users will have to log in to different GL instances, but then again, this is the purpose of SL multitenancy.
If you succeed we would appreciate it if you post your solution/feedback in this discussion.

Regards,
J.

**Reply by naimarshad** [reply]
Hi,
I think may be i was not clear to explain it. We have who solution hosted on domain (GL, SL, BBB). But users coming from different domains such as @gmail.com @org.com  etc. What we are looking here if user from @gmail.com wants to join he should land into a bbb instance bbb-01.meetings.com while users from @org.com will land on bbb-02.meetings.com.

Secondly thanks for explaining, can you point me to some documentation for tenant configuration. 

Regards,

**Comment by JeanPluzo** [comment]
Hi,
docs for tenant configuration are [right there](https://github.com/blindsidenetworks/scalelite/blob/master/docs/rake-README.md).
Now I understand what you're trying to do. It seems to me you're trying to filter users before they even sign up/sign in to GL. This might be too complicated for GL or BBB. An Idea: use a landing page where a user signs up. Once this user enters the e-mail address send him/her to the corresponding GL instance. The landing page could be a simple html page. Otherwise you'll have to build a filtering mechanism into GL. For that... can't really say how.

Regards,
J.
